markdown/en/documentation/tracking-with-js.md

```diff
@@ -0,0 +1,145 @@
+## Tracking with JavaScript
+
+Instead of using the built-in tracking you are also able to use our
+JavaScript-API to send your tracking events.
+
+___
+**IMPORTANT**
+
+**This documentation is NOT a comprehensive [FACT-Finder Tracking Documentation](https://doku.fact-finder.de/endoc/latest/fact-finder-integration/tracking-interface-integration). Please refer to the official FACT-Finder documentation to read more about the tracking itself.**
+___ 
+
+ 
+
+The [`Tracking12`](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/master/dist/ff-core.d.ts#L747) class contains the relevant tracking 
+methods all of which take one object as a parameter.
+
+You can create instances like:
+```Javascript
+const track = new factfinder.communication.Tracking12();
+```
+
+ 
+
+**INFO**
+
+For more technical information about this class or to get type completion support in many IDEs take a look 
+at our [ff-core.d.ts](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/master/dist/ff-core.d.ts) at [Github](https://github.com/FACT-Finder-Web-Components).
+
+### Retrieving necessary information
+Some information is provided by Web Components and or FACT-Finder. 
+
+
+#### sid
+You can retrieve the current FACT-Finder Web Components **sid** by calling this helper function: 
+
+`factfinder.common.localStorage.getItem("ff_sid");`
+
+**NOTE** You should always rely on this function for Safari private mode compatibility!
+
+
+#### id 
+The **id** parameter has to match the field value of the field configured as `trackingProductNumber ` [field role](https://web-components.fact-finder.de/documentation/field-roles)
+
+
+#### masterId
+The **masterId** parameter has to match the field value of the field configured as `masterArticleNumber` [field role](https://web-components.fact-finder.de/documentation/field-roles)
+
+
+#### channel
+The channel parameter can be retrieved by the `<ff-communicatrion>` element like: `var channel =document.querySelector("ff-communication").channel;` or by the 
```

Comment:
Typo: `<ff-communicatrion>`
This `var channel` thing could be formatted more nicely.